corehq/couchapps/attachments/views/attachments/map.js

```diff
@@ -10,6 +10,13 @@ function (doc) {
                     attachments[key] = doc._attachments[key];
                 }
             }
+            for (var key in doc.external_blobs) {
```

Comment:
Seems like this will throw an exception if `doc.external_blobs` is undefined, and same above for `doc._attachments`... I think that means I need to fix this and reindex staging again. Can someone sanity check me on that, please?